FinalEngine.Audio.OpenAL/OpenALSound.cs

```diff
@@ -7,15 +7,37 @@ namespace FinalEngine.Audio.OpenAL;
 using System;
 using CASLSound = CASL.Sound;
 
+/// <summary>
+/// Provides an OpenAL implementation of an <see cref="ISound"/>.
+/// </summary>
+/// <seealso cref="ISound" />
 public class OpenALSound : ISound
 {
+    /// <summary>
+    /// The sound instance for this <see cref="OpenALSound"/>.
+    /// </summary>
     private CASLSound? sound;
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="OpenALSound"/> class.
+    /// </summary>
+    /// <param name="filePath">
+    /// The file path of the audio source.
+    /// </param>
     public OpenALSound(string filePath)
```

Comment:
This is an issue but for now is okay. Not sure if this will be fixed: https://github.com/KinsonDigital/CASL/issues/158

Reply:
This is okay as this PR is only addressing documentation concerns.